addons/mass_mailing/models/mailing.py

```diff
@@ -330,6 +331,11 @@ def _compute_reply_to(self):
             elif mailing.reply_to_mode == 'update':
                 mailing.reply_to = False
 
+    @api.depends('mailing_model_id', 'mailing_domain')
+    def _compute_mailing_filter_count(self):
+        for record in self:
+            record.mailing_filter_count = self.env['mailing.filter'].search_count([('mailing_model_id', '=', record.mailing_model_id.id)])
```

Comment:
pro tips: never do search query in a loop

instead you can do a `read_group`, group by mail_id and then parse the result in the loop

Reply:
This field is supposed to be in the form view only. But indeed it can be used somewhere else in the future so I will make the changes 👍🏼